front/src/WebRtc/MediaManager.ts

```diff
@@ -13,7 +13,7 @@ if(localValueVideo){
 let videoConstraint: boolean|MediaTrackConstraints = {
     width: { min: 640, ideal: 1280, max: 1920 },
     height: { min: 400, ideal: 720 },
-    frameRate: {exact: valueVideo, ideal: valueVideo},
+    frameRate: { ideal: valueVideo },
```

Comment:
In RC3 we had just ```frameRate: valueVideo,```, not as an object. It worked, but seemed to have similar issues like people are reporting in comments.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/ConstrainULong
> In addition, you can specify the value as a simple long integer value, in which case the user agent does its best to match the value once all other more stringent constraints are met.

Not sure if interpreting this correctly, but this seems like passing a plain `Number` here is equivalent to passing an object with just the `ideal` specified. I chose the latter to match other constraints specified already.